spec/policies/membership_policy_spec.rb

```diff
@@ -21,13 +28,40 @@
     end
   end
 
+  describe 'approval?' do
+    context 'when person has a membership in the community' do
+      it 'allows access' do
+        expect(MembershipPolicy.new(person, nil).approval?).to be true
```

Comment:
Blocking: We should not allow approval actions when the person is merely a member and not a moderator.